### ISSUE TITLE
Expiring certificate was flagged one day after official check_http Nagios plugin did so

### ISSUE THREAD

**Issue by atc0005** [atc0005/check-cert#11]
We had a deprecated cert expiration check trigger last night at 7 pm (I think it was). This is the output for that deprecated check:

> expires in 30 day(s) (Sat 01 Aug 2020 12:59:00 AM CDT)

Output for the new check (specific to the leaf cert):

> expires next (on 2020-07-31 23:59:59 +0000 UTC)

and from the "Detailed Info" block:

> Expiration: 2020-07-31 23:59:59 +0000 UTC

I suspect that all time/date comparisons are being performed in UTC already, but I want to double-check and ensure that is the case. If so, I'm not too concerned (based on my current understanding) that the old check is in local time and the new one (presumably) is in UTC. 

In short: I'd want to have the comparison time zones match, so if the cert expiration is in UTC, then I think we should compare based on UTC.
